pkg/controller/forwarding/forwarder_controller.go

```diff
@@ -31,30 +27,30 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileForwarding{client: mgr.GetClient(), scheme: mgr.GetScheme()}
+	return &ReconcileForwarder{client: mgr.GetClient(), scheme: mgr.GetScheme()}
```

Comment:
simply out of curiosity, why are we renaming this? It looks like a lot of changes in this pr are related to this renaming and it doesn't change functionality...

Reply:
CRD Kinds are supposed to be nouns according to the k8s convention doc. It turned out to be a bigger change than I expected, but then I was already into it so I kept going...

Reply:
Also I thought it might be important to distinguish the TP from the GA API, but since they are v1 and v1alpha1 that wasn't really necessary.

Reply:
We had long discussions about it, I think the main reasons are:
- clarity: we may have a NamespaceLogForwarder in future, so the ClusterLogForwarder is unambiguous if that happens.
- k8s convention says that these names should be nouns, so forwarder instead of forwarding